README.md

```diff
@@ -1,5 +1,20 @@
 # Firefly
 
+## Intro
+
+Firefly is an open-source web-based UI library for astronomical data archive access and visualization developed in Infrared Processing and Analysis Center (IPAC) at [Caltech](https://caltech.edu).
+The development started with the Spitzer Heritage Archive (SHA), and continued in the WISE Image Archive, Planck Image Archive, and other web applications in the NASA/IPAC Infrared Science Archive (IRSA) applications. It was open sourced in 2015, hosted at GitHub.
+
+Firefly is designed to build complex web-based front-end application making use of  reusable [React](https://reactjs.org) components to enable access and explore astronomical data with advanced data visualization capabilities.
+
+The visualization provides user with an integrated experience with brushing and linking capabilities among images, catalogs, and plots. Firefly is used in NASA/IPAC Infrared Science Archive ([IRSA](https://irsa.ipac.caltech.edu)) GUIs to query and visualize data from missions such as WISE, Spitzer, SOFIA, ZTF, PTF, etc. and a large number of highly-used contributed data products from a diverse set of astrophysics projects, in Rubin Observatory Science Platform Portal, at NASA Exoplanet Science Institute ([NExScI](https://nexsci.caltech.edu)) and in NASA/IPAC Extragalactic Database ([NED](http://ned.ipac.caltech.edu)).
```

Comment:
"The visualization components provide a user with ..."

"... IRSA ... from a diverse set of astrophysics projects."  Then "It is also used in various user interfaces of the NASA Exoplanet Science Institute ([NExScI](https://nexsci.caltech.edu)) and the NASA/IPAC Extragalactic Database ([NED](http://ned.ipac.caltech.edu)), as well as to construct the Portal Aspect of the Vera C. Rubin Observatory Science Platform."